server/fleet/software.go

```diff
@@ -2,9 +2,20 @@ package fleet
 
 import "time"
 
-type SoftwareCVE struct {
+type CVE struct {
 	CVE         string `json:"cve" db:"cve"`
-	DetailsLink string `json:"details_link" db:"details_link"`
+	DetailsLink string `json:"details_link" db:"-"`
+	// These are double pointers so that we can omit them AND return nulls when needed.
+	CVSSScore        **float64 `json:"cvss_score,omitempty" db:"cvss_score"`
+	EPSSProbability  **float64 `json:"epss_probability,omitempty" db:"epss_probability"`
+	CISAKnownExploit **bool    `json:"cisa_known_exploit,omitempty" db:"cisa_known_exploit"`
+}
+
+type CVEScore struct {
+	CVE              string   `db:"cve"`
+	CVSSScore        *float64 `db:"cvss_score"`
+	EPSSProbability  *float64 `db:"epss_probability"`
+	CISAKnownExploit bool     `db:"cisa_known_exploit"`
```

Comment:
Why are `CVSSScore` and `EPSSProbability` pointers but not `CISAKnownExploit`?

Reply:
fixed in c46ba026a